source/includes/logging-admonition.rst

```diff
@@ -0,0 +1,11 @@
+.. important::
+    
+   To output logs, you must specify both a logfile and the ``OutputInternalLogsToFile`` option. 
+
+   Use the following sample configuration to make the {+product+} output information to a logfile:
+
+   .. code-block:: json
+
+      OutputInternalLogsToFile: true,
+      LogFileName: "<path to your logfile>"
```

Comment:
```suggestion
      "OutputInternalLogsToFile": true,
      "LogFileName": "<path to your logfile>"
```
I recommend making it actual JSON so one does not need to add quotes after copy-pasting.

Reply:
Great catch!